### PR TITLE
GC features pattern update and fixes build

### PR DESCRIPTION
**Pull request by GormFrank**
Fixes build required to bump WET-BOEW dependency to 4.0.33. 
Relates to #1628 

**(GC features now need documentation to track pattern updates over the history of GCWeb.)**

### CODE REVIEW

**Comment by GormFrank** [comment]
Documentation is included and ready for review.

**Comment by GormFrank** [comment]
> Can you document the pattern and the data schema of the feature.
> 
> :-)

Documentation is ready for review.

File: site/pages/component/gc-features-en.hbs [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+	rdfs:label "Features link"@en ;
+	rdfs:label "Lien en vedette"@fr ;
+
+comp:features
+	dct:issued "2017-01-01"^^xsd:date ;
+	a rdf:Property ;
+	rdfs:domains comp:GcFeatures ;
+	rdfs:range comp:Feature .
+
+comp:gcFeaturesCol
```
Comment (duboisp) [inline site/pages/component/gc-features-en.hbs]: Not need to prefix. it is already under vocab.canada.ca. It is already implicit

File: site/pages/component/gc-features-en.hbs [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+<h4>JSON-LD sample</h4>
+<pre><code>{
+	"@context": {
+		"foaf": "http://xmlns.com/foaf/0.1/",
+		"dct": "http://purl.org/dc/terms/",
+		"comp": "http://vocab.canada.ca/wms/component"
+		"html": "http://vocab.canada.ca/wms/html/img"
+	},
+	"dct:title": "Government initiatives",
+	"comp:srvInfoGroupAs": "comp:GcFeatures",
```
Comment (duboisp) [inline site/pages/component/gc-features-en.hbs]: Ensure the JSON-LD sample is conforming to the Shape graph

File: site/pages/component/gc-features-en.hbs [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+		<div class="col-lg-4 col-md-6 mrgn-bttm-md">
+			<h3 class="h5"><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
+			<img src="{{assets}}/img/360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
+			<p>Brief description of the feature being promoted.</p>
+		</div>
+	</div>
+</section>
+
+<h3>GC activities and initiatives - v6.1</h3>
+<section class="gc-initiatives">
```
[the remark this answers is not in the document]
Comment (duboisp) [reply site/pages/component/gc-features-en.hbs]: To be fixed later

File: site/pages/component/gc-features-en.hbs [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+
+comp:featureCol
+	dct:issued "2017-01-01"^^xsd:date ;
+	rdfs:label "Column layout rendering"@en ;
+	a rdf:Property ;
+	rdfs:range comp:Feature1Col ;
+	rdfs:range comp:Feature2Col ;
+	rdfs:range comp:Feature3Col .
+
+comp:Feature1Col
```
[the remark this answers is not in the document]
Comment (duboisp) [reply site/pages/component/gc-features-en.hbs]: To be fixed later, don't impact currently build

**Pull request merged**